qutip/tests/test_propagator.py

```diff
@@ -127,5 +127,29 @@ def testPropHDims():
     U = propagator(H,1, unitary_mode='single')
     assert_equal(U.dims,H.dims)
 
+
+def testPropHSuperWithoutCops():
+    "Propagator: super operator without collapse operators"
+    H = tensor(sigmaz(), qeye(2))
+    H = to_super(H)
```

Comment:
Why `to_super` and not `liouvillian`?

Reply:
Because I did not think to use `liouvillian`. Fixed. :)